src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+public abstract class Figure implements Informable, AreaCounter {
+    private String color;
+    private String name;
+    private double area;
```

Comment:
lets remove this field; We will obtain area dynamically through countArea() method